README.md

```diff
@@ -148,6 +163,41 @@ Additionally, unattended-upgrades support two macros (variables), derived from `
 
 Using `${distro_codename}` should be preferred over using `stable` or `oldstable` as a selected, as once `stable` moves to `oldstable`, no security updates will be installed at all, or worse, package from a newer distro release will be installed by accident. The same goes for upgrading your installation from `oldstable` to `stable`, if you forget to change this in your origin patterns, you may not receive the security updates for your newer distro release. With `${distro_codename}`, both cases can never happen.
 
+## Systemd timers
+
+Documentation for systemd/Timers: <https://wiki.archlinux.org/title/systemd/Timers>
+
+### Debian Default Configuration
+
+* Download daily between at 6am AND 6pm
+* Install daily between 5am - 7pm
+
+```yaml
+unattended_systemd_timer_override: false # (default)
+# apt-daily timer
+unattended_apt_daily_oncalendar: "*-*-* 6,18:00" # (default)
+unattended_apt_daily_randomizeddelaysec: "12h" # (default)
+# apt-daily-upgrade timer
+unattended_apt_daily_upgrade_oncalendar: "*-*-* 6:00" # (default)
+unattended_apt_daily_upgrade_randomizeddelaysec: "60m" # (default)
+```
+
+### Override Example
+
+* Download is between 12:20am - 12:40am
```

Comment:
this seems wrong to me. According to the docs, the RandomizedDelaySec should only be added.